src/js/api.js

```diff
@@ -53,7 +53,7 @@ export function sendEvent(elementId, eventType) {
 export function registerSession(userData) {
   const jsonObjectPromise = prepareSessionData(userData);
 
-  jsonObjectPromise.done(jsonObject => $.ajax({
+  jsonObjectPromise.done((jsonObject) => $.ajax({
```

Comment:
Why the added parentheses in arrow function with only 1 parameter?

Reply:
This is connected with new AirBnb eslint. They recommend adding parentheses to all arrow functions ([link](https://github.com/airbnb/javascript#arrows--one-arg-parens)).